src/backend/distributed/Makefile

```diff
@@ -90,11 +90,11 @@ install: cleanup-before-install
 
 # install and install-downgrades should be run sequentially
 install-all: install
-	make install-downgrades
+	$(MAKE) install-downgrades
```

Comment:
If you ran `make -sj8 install-all` or `make --silent --jobs=8 install-all` on the root directory, some of the extra options will be lost.

`$(MAKE)` allows make to pass some flags to a sub-make command. You can test `make --just-print install-all`  on master and verify that it id more than just printing the downgrade scripts, and actually installed them because the `--just-print` flag was lost.

We can also use a `+` token in the beginning of the line, however that is not as good for readability IMO.

Reply:
If you want to learn more: https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html